Commands/youtube.py

```diff
@@ -0,0 +1,115 @@
+from discord.ext import commands, tasks
+from .GuildData import get_all_guilds_data, get_guild_data, GuildData
+from Helpers.parser import parse_channel
+from Helpers.log import info, fatal
+from Translations.Translations import get_text as translate
+from Helpers.TranslationHelper import get_culture_from_context as culture
+import discord
+import requests
+import os
+
+from Scheduler.Scheduler import get_latest_video
+
+
+class Youtube(commands.Cog, name="Youtube_lists"):
+
+    @commands.command(
+        name="add_youtube",
+        usage="add_youtube_usage",
+        help="add_youtube_help",
+    )
+    async def add_youtube_channel(
+        self, ctx: commands.Context, youtube_channel_id: str, text_channel: str
+    ):
+        """
+        Add a Youtube channel to be notified
+        :param ctx: The current context. (discord.ext.commands.Context)
+        :param youtube_channel_id: The Youtube channel to be notified of (str)
+        :param text_channel: The text channel that will receive the notification (str)
+        """
+        guild_data = await get_guild_data(ctx.message.guild.id)
+        # Error if not admin
+        if not guild_data.user_is_admin(ctx.author):
+            gif = translate("not_admin_gif", await culture(ctx))
+            return await ctx.send(gif)
+
+        text_channel = text_channel.lower()
+
+        # Sanitize channel name
+        text_channel = parse_channel(text_channel)
+
+        # TODO: throw specific error with message when channel ID is wrong
+        latest_video = await get_latest_video(youtube_channel_id)
+
+        # Retrieve channel
+        channel = ctx.bot.get_channel(int(text_channel))
+        if not channel:
+            channel = discord.utils.get(ctx.channel.guild.channels, name=text_channel)
+        
+        # TODO: Give information to the user when the text channel does not exist
+        if not channel:
+            raise Exception("Invalid text channel provided")
```

Comment:
This will leave a "da kennek nie foump" generic error message in Discord, no, if only handled by an Exception?